test/image.cpp

```diff
@@ -496,9 +496,11 @@ void static_checks() {
     BOOST_STATIC_ASSERT(view_is_mutable<rgb8_planar_view_t>::value);
 
     BOOST_STATIC_ASSERT((boost::is_same<derived_view_type<cmyk8c_planar_step_view_t>::type, cmyk8c_planar_step_view_t>::value));
-    BOOST_STATIC_ASSERT((boost::is_same<derived_view_type<cmyk8c_planar_step_view_t, bits16, rgb_layout_t>::type,  rgb16c_planar_step_view_t>::value));
+    BOOST_STATIC_ASSERT((boost::is_same<derived_view_type<cmyk8c_planar_step_view_t, std::uint16_t, rgb_layout_t>::type,  rgb16c_planar_step_view_t>::value));
     BOOST_STATIC_ASSERT((boost::is_same<derived_view_type<cmyk8c_planar_step_view_t, use_default, rgb_layout_t, mpl::false_, use_default, mpl::false_>::type,  rgb8c_step_view_t>::value));
 
+    static_assert(boost::is_same<derived_view_type<cmyk8c_planar_step_view_t, std::uint16_t, rgb_layout_t>::type,  rgb16c_planar_step_view_t>::value, "XXXXXXXXXXXXXXXXXX");
+
```

Comment:
You can surely come up with a more informative (and eloquent) message here, right ? ;-) )

Reply:
Shoot, that `static_assert` sneaked by mistake! I will remove it.

I used it as backup check when I was hitting some mysterious compilation errors during the refactoring. (I don't like `BOOST_STATIC_ASSERT` obscurity :-))